### DISCUSSION TITLE
ant-desigm-pro我觉得可以增加一个单元测试

### DISCUSSION THREAD

**Discussion by JackDan9** [ant-design/ant-design#32856] [Ideas]
[ant-design-pro](https://github.com/ant-design/ant-design-pro)是否可以维护一个单元测试和e2e测试进去了，因为社区的代码更多回去参照ant-design-pro的代码，个人觉得可以维护一个单元测试，当然我也愿意参与和维护这块了，但是大佬们看看是否需要，给一些建议，感谢。

**Comment by afc163** [comment]
是有的，不过实现的比较一般，欢迎来贡献

https://github.com/ant-design/ant-design-pro/blob/3e5903491b09a70b5757ad6fc6b40b9259f592b5/package.json#L38-L40

**Reply by JackDan9** [reply]
感谢！那我开始参与了。
